data/wanderer/wanderers start.txt

```diff
@@ -465,6 +466,19 @@ mission "Wanderers: Outfit License Patch"
 
 
 
+# Compatibility patch for pilots that did the above missions before 0.10.9.
+mission "Wanderer Technology Patch"
```

Comment:
Given that this is about planetary facilities, I don't think "technology" is a good name.